source/AI.cpp

```diff
@@ -3583,11 +3601,11 @@ double AI::RendezvousTime(const Point &p, const Point &v, double vp)
 	// to intersect the target?
 	// (p.x + v.x*t)^2 + (p.y + v.y*t)^2 = vp^2*t^2
 	// p.x^2 + 2*p.x*v.x*t + v.x^2*t^2
-	//    + p.y^2 + 2*p.y*v.y*t + v.y^2t^2
-	//    - vp^2*t^2 = 0
+	//	+ p.y^2 + 2*p.y*v.y*t + v.y^2t^2
+	//	- vp^2*t^2 = 0
 	// (v.x^2 + v.y^2 - vp^2) * t^2
-	//    + (2 * (p.x * v.x + p.y * v.y)) * t
-	//    + (p.x^2 + p.y^2) = 0
+	//	+ (2 * (p.x * v.x + p.y * v.y)) * t
+	//	+ (p.x^2 + p.y^2) = 0
```

Comment:
This looks like a mistake - why are you adding whitespace in a comment about a section that you aren't changing?

Reply:
....oh, whoops. I think that was an accidental consequence of doing a global search & replace for <4 spaces> to <tab>, because I'd made enough edits to AI.cpp before I realized my IDE was still defaulting to spaces that it seemed quickest.

If it's desired, I'm happy to revert that change.